meetings/2023-02-02.md

```diff
@@ -0,0 +1,210 @@
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* SC
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: Keeping the topics focused on work we need to iron out for the next release.
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback. Links to servers/applications and demos welcome.
+
+
+
+### Add ldn-channel-2023 ED
+URL: https://github.com/solid/notifications/pull/147
+
+* SC: There was rough consensus that some of the notions, e.g., `activityType` feature, and core `receiver`, could move to Notifications Protocol.
+* SC: Proposal to add `activityType` to Notifications Protocol.
+* SC: Proposal to add `receiver` to Notifications Protocol.
+* SC: 201 is required by LDNChannel2023. It would only be optional on the Notifications Protocol in addition to 200 in that if a SubscriptionServer were to return 201, it'll need to include `Location`.
+* SC: Also taking https://github.com/solid/notifications/pull/147#issuecomment-1410518467 into account - similar to the line of thing from last weeks recommendation on `filter`.
+* SC: Should we push e.g., filtering concept/requirements through Version 0.2.0 or wait for later?
+* RG: Some repetition of content between the main protocol and ldn-channel-2023.
+* eP: Nothing stops us from pushing an updated one. Prefer to track an issue how we want to add it to the protocol, e.g., inline in LDN for example. That is discussed.
+* AC: If we can't release quickly we should try to get things quickly but if we can do it with more cadence then okay.
+* SC: ...
+* eP: can we clarify for every document who has to approve it. Team has to approve every version of every spec or major changed? Also whoever approved there is a change-log. Going from 0.2 to 0.3 should be a simpler change-log. The review of that shouldn't take too long. I propose to make changes small.
+* RG: can you clarify how much difference / changes to expect in the main protocol as with the LDN protocol. I don't think there'd be many.
+* eP: At least the filtering stuff we were discussing or receiver.. It can go into 0.2 or 0.3. The diff is small. Currently the person has to read the whole doc.
+* SC: I wonder if we can just push it in 0.2. On one hand we had very little discussion and implementation experience. On the other hand we feel really confident.
```

Comment:
```suggestion
* RG: can you clarify how much difference / changes to expect in the main protocol as with the LDN protocol? I don't think there'd be many.
* eP: At least the filtering stuff we were discussing or receiver... It can go into 0.2 or 0.3. The diff is small. Currently, the person has to read the whole doc.
* SC: I wonder if we can just push it in 0.2. On one hand, we had very little discussion and implementation experience. On the other hand, we feel really confident.
```